src/components/fields/date-range-field/date-range-field.tsx

```diff
@@ -26,7 +26,14 @@ export const DateRangeField = (props: IGenericFieldProps<TDateRangeFieldSchema>)
 		id,
 		isDirty,
 		onChange,
-		schema: { dateFormat = DEFAULT_DATE_FORMAT, label: _label, validation, variant, ...otherSchema },
+		schema: {
+			dateFormat = DEFAULT_DATE_FORMAT,
+			label: _label,
+			validation,
+			variant,
+			hideInputKeyboard,
```

Comment:
no need to extract `hideInputKeyboard`, you can leave it in `otherSchema` and it will be applied to `Form.DateRangeInput`